src/meinberlin/meinberlin/static/js/Packages/Meinberlin/IdeaCollection/Process/Process.ts

```diff
@@ -25,6 +25,43 @@ import * as SIWorkflow from "../../../../Resources_/adhocracy_core/sheets/workfl
 
 var pkgLocation = "/Meinberlin/IdeaCollection/Process";
 
+var bindFacets = (
```

Comment:
The name is misleading. Either rename this to `getFacets()` or wrap it in a `scope.$watch()`.